lib/Common/Memory/RecyclerWriteBarrierManager.cpp

```diff
@@ -225,11 +231,6 @@ X64WriteBarrierCardTableManager::Initialize()
 
         LPVOID cardTableSpace = ::VirtualAlloc(NULL, _cardTableNumEntries, MEM_RESERVE, PAGE_READWRITE);
 
-        if (cardTableSpace == nullptr)
```

Comment:
This was a weird one :) if it returned null, we were returning false (which clang didn't like)- if we remove the check, we'd return null anyway, so the check was unnecessary